gui/esp.cpp

```diff
@@ -0,0 +1,118 @@
+#include "pch-il2cpp.h"
+#include "esp.hpp"
+#include "DirectX.h"
+#include "utility.h"
+#include "game.h"
+
+#include <../includes/imgui/imgui_internal.h> // TODO : Fix this dir mess
```

Comment:
fix this

Reply:
this can more easily be fixed with premake or cmake. Would it be appreciated if i changed the project to premake? (personal favourite of the two since its easier to use for most people)

Reply:
We already have includes\ as an additional include directory in the properties.  Just change it to

`#include <imgui/imgui_internal.h>`